streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamImplTest.java

```diff
@@ -2330,4 +2333,101 @@ public void shouldNotAllowNullNamedOnProcessWithStores() {
         assertThat(exception.getMessage(), equalTo("named can't be null"));
     }
 
+
+    @Test
+    public void shouldKTableFromKStream() {
```

Comment:
This test seems to be unnecessarily complex. If we want to test the new `toTable()` operator, we should just do it directly and not by using an additions `builder.table().toStream()` construct...